src/fromager/commands/build.py

```diff
@@ -127,13 +127,24 @@ def build_sequence(
                 wheel_filename = _build(
                     wkctx, resolved_version, req, source_download_url
                 )
+            packages_built.append(entry)
 
             server.update_wheel_mirror(wkctx)
             # After we update the wheel mirror, the built file has
             # moved to a new directory.
             wheel_filename = wkctx.wheels_downloads / wheel_filename.name
             print(wheel_filename)
 
+    logger.info("\nTransaction summary:")
```

Comment:
How hard would it be to get this into a separate file?

Reply:
Probably just adding a file handler